TangoRosStreamer/app/src/main/java/eu/intermodalics/tango_ros_streamer/activities/RunningActivity.java

```diff
@@ -603,7 +648,7 @@ public void onError(Throwable t) {
 
     private void restartTango() {
         if (mParameterNode != null) mParameterNode.setPreferencesFromParameterServer();
-        updateSaveMapButton();
+        updateLoadAndSaveMapButtons();
```

Comment:
This function is called: `restartTango`, so in this case `onTangoStatus` will be called.
Could you move the function call `updateLoadAndSaveMapButtons` to `onConfigurationChanged` instead?